src/services/business-lookup-services.ts

```diff
@@ -30,10 +30,10 @@ export default class BusinessLookupServices {
   /**
    * Searches for business by code or words.
    * @param query code or words to search
-   * @param status status to match (ACTIVE OR HISTORICAL or '' to match all statuses)
+   * @param status status to match (ACTIVE or HISTORICAL or '' to match all statuses)
    * @returns a promise to return the search results
    */
-  static async search (query: string, status = 'ACTIVE'): Promise<BusinessLookupResultIF[]> {
+  static async search (query: string, status: string): Promise<BusinessLookupResultIF[]> {
```

Comment:
The BusinessLookup component calls this and always includes the status parameter so I made it required (not optional).